x-pack/legacy/plugins/actions/server/builtin_action_types/webhook.ts

```diff
@@ -64,6 +86,11 @@ async function executor(execOptions: ActionTypeExecutorOptions): Promise<ActionT
   const { user: username, password } = execOptions.secrets as ActionTypeSecretsType;
   const { body: data } = execOptions.params as ActionParamsType;
 
+  if (!configurationUtilities.isWhitelistedHostname(url)) {
+    log(`warn`, `error on ${id} webhook event: The target "${url}" ahs not been whitelisted`);
```

Comment:
probably should mention "action" in there somewhere?  and s/ahs/has/

Reply:
cleaned up all the messaging making it clear what the source of the error is.